dashboards/kubernetes_capacity_planning/README.md

```diff
@@ -0,0 +1,25 @@
+# Kubernetes Capacity Planning
```

Comment:
I would use `Kubernetes` consistently in the README, no lowercase neither k8s. 

Also VPA acronym should be expanded to "Vertical Pod Autoscaler" as not everyone might be familiar with it.